src/components/MagicCodeInput.tsx

```diff
@@ -371,75 +364,63 @@ function MagicCodeInput(props) {
                             onLayout={(e) => {
                                 inputWidth.current = e.nativeEvent.layout.width;
                             }}
-                            ref={(ref) => (inputRefs.current = ref)}
-                            autoFocus={props.autoFocus}
+                            ref={(inputRef) => (inputRefs.current = inputRef)}
+                            autoFocus={autoFocus}
                             inputMode="numeric"
                             textContentType="oneTimeCode"
-                            name={props.name}
-                            maxLength={props.maxLength}
+                            name={name}
+                            maxLength={maxLength}
                             value={input}
                             hideFocusedState
-                            autoComplete={input.length === 0 && props.autoComplete}
-                            shouldDelayFocus={input.length === 0 && props.shouldDelayFocus}
+                            autoComplete={input.length === 0 ? autoComplete : undefined}
+                            shouldDelayFocus={input.length === 0 && shouldDelayFocus}
                             keyboardType={CONST.KEYBOARD_TYPE.NUMBER_PAD}
-                            onChangeText={(value) => {
-                                onChangeText(value);
+                            onChangeText={(textValue) => {
+                                onChangeText(textValue);
                             }}
```

Comment:
Can't it be just like this?
```suggestion
                            onChangeText={onChangeText}
```

Reply:
It can, thanks!